update_multiarch.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Not sure that change is safe?

Reply:
Sorry, was a fix for #137 and applied during testing .. and forgotten. Removed since that is out of scope for this PR of course. Fixed